content/reference/sql/server/troubleshooting.md

```diff
@@ -42,6 +42,11 @@ To run garbage collection, you should first stop your running server. It is not
 
 Online garbage collection is on the roadmap and we will be implementing it in the back half of 2022.
 
+Another potential cause is a commit-heavy workflow that uses a database design that is antagonistic to Dolt's structural sharing. We've written thoroughly about this [here](https://www.dolthub.com/blog/2020-05-13-dolt-commit-graph-and-structural-sharing/), but some examples include
+
+* Primary keys with random values
+* Adding a column to a table with a non-null default
```

Comment:
The non-null default is a red herring. We had an optimization to not fork storage until values were added to a column but that is gone in the new format. 

Just saying adding a column to a table forks the storage for that table. Dolt is row major and builds chunks for each primary key, row values pair. The row values encodes the schema length so every row now requires a new chunk.